azure-pipelines.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       pip install pytest pytest-cov pytest-mock pytest-timeout pytest-azurepipelines
       pip install -e .
       export NREL_API_KEY=$(nrelApiKey)
+      echo $NREL_API_KEY
```

Comment:
not a big deal in this instance but it defeats the purpose of a secret key. the CI logs are available for all to see.

Reply:
I played with this on my fork and noticed that Azure blocked it out as "**". Are you able to see it?

Reply:
I think Travis and other CI will not show any secrets that have the word "SECRET" or "PASSWORD" in it, **but I wouldn't count on it**

Reply:
@CameronTStark no I see [`/home/vsts/work/_temp/0cb00519-40af-46ec-994a-cab3c0d8db0c.sh: line 2: nrelApiKey: command not found
`](https://dev.azure.com/solararbiter/pvlib%20python/_build/results?buildId=2533&view=logs&j=ea977d73-b8d6-59cf-b2df-c65a31e93ffc&t=2690bf14-3208-5a31-bf74-aa7a625c04e8&l=18) instead of `*`. Nice to know that there is some built in protection but like @mikofski says I wouldn't count on it.

Reply:
[Here's a link to the pipeline run for my fork on the same commit where it outputs "***" for the key.](https://dev.azure.com/ctstark/pvlib-test/_build/results?buildId=55&view=logs&j=83ed75f2-3f6e-51e0-dc7b-dfceecefbbcd&t=0b9cc177-13eb-5a9e-870b-984e27ba97f8&l=81) It is printed like that in the raw output too so I think it's safe. I'm aware of the security implications and wouldn't do this for a more valuable key. I tested this out on my own fork initially with a dummy variable and witnessed the "***" for that before putting in the actual key. 

A quirk of this secret variable is that it is sheltered from the Python code even though its in the environment. That's why it has be renamed because I calling `os.environ['nrelApiKey']` didn't work.

Reply:
Interesting. I get a 401 not authorized error when I try your link.

Reply:
> Interesting. I get a 401 not authorized error when I try your link.

Yes, I've been getting that for pvlib-python's Azure links as well for an hour or so. Did permissions change for it?

Reply:
I didn't change anything.

Reply:
If I logout then I can see it. Strange...